constraint_utils/constraint_basics.py

```diff
@@ -191,12 +191,32 @@ def split_context_mt(raw_context_ht_path: str, coverage_ht_paths: Dict[str, str]
 
 def pre_process_data(ht: hl.Table, split_context_ht_path: str,
                      output_ht_path: str, overwrite: bool = False) -> None:
+    """ Joins the input hail table and context table, drop 'a_index', 'was_split', and 'colocated_variants' columns
```

Comment:
```suggestion
    """
    Join gnomAD and VEP context Tables.
    
    Function drops `a_index`, `was_split`, and`colocated_variants` annotations from gnomAD data.
   
```

in general, we try to format docstrings like this on the gnomad methods team:
```
"""
Description (this should be a command)

:param x: Description
:return: Description of what function returns
"""
```
https://peps.python.org/pep-0257/